sentinel-adapter/sentinel-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/AbstractDubboFilter.java

```diff
@@ -43,15 +43,24 @@ protected String getResourceName(Invoker<?> invoker, Invocation invocation) {
         return buf.toString();
     }
 
-    protected String getResourceName(Invoker<?> invoker, Invocation invocation, String prefix) {
+    protected String getMethodResourceName(Invoker<?> invoker, Invocation invocation, String prefix) {
         if (StringUtil.isBlank(prefix)) {
-            return getResourceName(invoker, invocation);
+            return getMethodResourceName(invoker, invocation);
         }
         StringBuilder buf = new StringBuilder(64);
         return buf.append(prefix)
-                .append(getResourceName(invoker, invocation))
-                .toString();
+            .append(getMethodResourceName(invoker, invocation))
+            .toString();
+    }
 
+    protected String getInterfaceName(Invoker<?> invoker) {
+        return invoker.getInterface().getName();
+    }
 
+    protected String getInterfaceName(Invoker<?> invoker, String prefix) {
+        if (StringUtil.isBlank(prefix)) {
+            return getInterfaceName(invoker);
+        }
+        return prefix + getInterfaceName(invoker);
```

Comment:
It seems that normally when overloading methods, the method which has less parameters call the method which has more. This is the opposite, but it is okay.

Reply:
Besides, I found that dubbo-adapter of 2.7.x has a `useGroupAndVersion` setting.
`String getMethodResourceName(Invoker<?> invoker, Invocation invocation, Boolean useGroupAndVersion)`.
`DubboConfig.getDubboInterfaceGroupAndVersionEnabled(), csp.sentinel.dubbo.interface.group.version.enabled`
Do we need to add this in 2.6.x?

Another setting is `csp.sentinel.dubbo.trace.biz.exception.enabled` in 2.7.x, and it is unused.

Reply:
> Besides, I found that dubbo-adapter of 2.7.x has a `useGroupAndVersion` setting.
> `String getMethodResourceName(Invoker<?> invoker, Invocation invocation, Boolean useGroupAndVersion)`.
> `DubboConfig.getDubboInterfaceGroupAndVersionEnabled(), csp.sentinel.dubbo.interface.group.version.enabled`
> Do we need to add this in 2.6.x?

Yes, we could add this in the future. We may open another issue to address this.

> Another setting is `csp.sentinel.dubbo.trace.biz.exception.enabled` in 2.7.x, and it is unused.

Actually I don't think it's useful.

Reply:
OK, I agree with you. IMHO the prefix is more useful, maybe enough at present.

Reply:
> OK, I agree with you. IMHO the prefix is more useful, maybe enough at present.

The group and version info can be useful. This could be contributed by community later.

Reply:
Thanks for detail description, group and version are useful indeed, got it : )